src/main/scala/com/databricks/labs/overwatch/utils/Structures.scala

```diff
@@ -124,13 +128,15 @@ case class ModuleStatusReport(
       status,
       recordsAppended,
       lastOptimizedTS,
-      vacuumRetentionHours
+      vacuumRetentionHours,
```

Comment:
it also makes sense to externalize vacuum - we don't need big cluster for it, so running it inside the same job cluster is just waste of resources.

Reply:
I assumed folks would just use an autoscaling cluster for this which would handle it right? This was one of the primary reasons for externalizing optimize was to enable a standard autoscaling cluster to be efficient.